help/destinations/catalog/personalization/pega.md

```diff
@@ -126,11 +130,11 @@ The customer audience membership data fields are added as predictors in Adaptive
 
 ## Additional resources {#additional-resources}
```

Comment:
Use the same additional resources phrasing as the pega-v2 page.

Reply:
updated